integration-cli/requirements.go

```diff
@@ -170,6 +170,16 @@ var (
 		},
 		"Test cannot be run when remapping root",
 	}
+	x86_64 = testRequirement{
+		func() bool {
+			out, err := exec.Command("uname", "-p").Output()
+			if err != nil && (strings.Contains(string(out), "x86_64") || strings.Contains(string(out), "i386")) {
```

Comment:
I think this test is poorly named, and that there's no reason why we should have to exclude tests in this way either at this point -- so far, we aren't using any images that can't really be replicated on other Linux platforms without too much trouble.